Resources/Prototypes/_Andromeda/Entities/Objects/Weapons/Guns/Ammunition/Magazines/light_rifle.yml

```
@@ -0,0 +1,96 @@
+# Magazines
+- type: entity
+  id: SmartgunMagazineLightRifleBox
+  name: "Smartgun box magazine (.30 rifle)"
+  parent: BaseMagazineLightRifle
+  components:
+  - type: Tag
+    tags:
+      - LightRifleHeavy
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 4 but found 6

(indentation)